### ISSUE TITLE
Platform not starting (LazyInitializationException)

### ISSUE THREAD

**Issue by stephanvane** [tolgee/tolgee-platform#2427]
**Describe the bug**
Since version `v3.70.3` the docker container is not starting with the default settings. Hibernate throws `org.hibernate.LazyInitializationException`

**To Reproduce**
run 
```
docker run -p 8085:8080 tolgee/tolgee:v3.70.3
```

In the console, the following error appears:

> org.hibernate.LazyInitializationException: Unable to perform requested lazy initialization [io.tolgee.model.UserAccount.emailVerification] - no session and settings disallow loading outside the Session

Or: go to http://localhost:8085, to see:

> {"code":"unexpected_error_occurred","params":["org.hibernate.LazyInitializationException"]}

**Expected behavior**
Version 3.70.2 and older run without any problems, I would expect the same behavior for newer versions

**Versions and environment**
- working: v3.70.2
- broken: v3.70.3

ps. neither version can be killed by `ctrl+c`. Only by running `docker kill <container-id>`

**Comment by JanCizmar** [comment]
Hello! You're right, it's weird that the tests pass. I am working on a fix.

**Comment by JanCizmar** [comment]
As a workaround, you can enable authentication via
```
tolgee:
  authentication:
    enabled: true
```

This only happens when authentication is disabled by default.

https://tolgee.io/platform/self_hosting/configuration#authentication

**Comment by stephanvane** [comment]
Seems that I 'beat' @fryette by 2 just minutes :)

Thank you for the quick response and workaround! 👍

**Comment by fryette** [comment]
With the latest changes I still experience same issue(if not using workaround)

**Comment by JanCizmar** [comment]
It should be fixed in the latest release (3.70.8).

**Comment by fryette** [comment]
Confirmed. Working as expected
